xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecFFmpeg.cpp

```diff
@@ -150,8 +150,7 @@ bool CDVDAudioCodecFFmpeg::AddData(const DemuxPacket &packet)
   AVPacket* avpkt = av_packet_alloc();
   if (!avpkt)
   {
-    CLog::Log(LOGERROR, "CDVDAudioCodecFFmpeg::{} - av_packet_alloc failed: {}", __FUNCTION__,
-              strerror(errno));
+    CLog::LogF(LOGERROR, "CDVDAudioCodecFFmpeg: av_packet_alloc failed: {}", strerror(errno));
```

Comment:
Before:
`CDVDAudioCodecFFmpeg::AddData ...`

After:
`AddData:CDVDAudioCodecFFmpeg: ...`

Order is reversed or I'm missing something?

Reply:
The PR is about updating the usage, not adjusting the actual messaging. Basically we just want to make the usage of `__FUNCTION__` obsolete and follow a standard practice.

Yes this changes the log messages a bit but individual components can be adjusted in the future. Adjusting each line by hand (~2500 of them) isn't really feasible so the scripting is the best option here.

Really if we care about locating the exact location we should be using `__FILE__` and `__LINE__` (which may be a solution in the future).

Reply:
Indeed, the process of updating individual components is that we merge this PR and let all logs going forward highlight what needs to change. From this point we should discover bugs in the logs, not the source.